.travis.yml

```diff
@@ -7,8 +7,11 @@ matrix:
         - brew install gnu-tar
       script: make test
     - os: linux
-      sudo: false
+      dist: trusty
+      sudo: true
```

Comment:
We don't need `sudo`, but [adding it triggers use of a GCE VM](https://docs.travis-ci.com/user/trusty-ci-environment#Overview) rather than a Docker container, and the container took >10 minutes longer to finish a build.